### DISCUSSION TITLE
.pth模型转换为.pt模型

### DISCUSSION THREAD

**Discussion by itfish** [STVIR/pysot#146] [General]
大神你好，能否给出.pth模型转换为.pt模型的代码，我总是转换不成功，如能帮助，将非常感谢！！！

**Comment by lb1100** [comment]
what is `.pt`？

**Comment by itfish** [comment]
我想用libtorch C++语言加载模型，但是libtorch加载.pt模型，需要将.pth预训练模型转换为.pt模型

**Comment by JensenHJS** [comment]
hello itfish,do you solve the problem?  Load the model with libtorch C++.用libtorch C++语言加载模型
Looking for your reply

**Comment by mqingkui** [comment]
@JensenHJS 请问，你搞定这个pth转pt的工作了吗？我也卡在这里了，track那块效果不对，我们可以交流下吗

**Comment by JensenHJS** [comment]
不好意思，没有解决，你可以看下这个xurui/SiamRPNTracker，它这个是libtorch里c++推理。

**Comment by mqingkui** [comment]
@JensenHJS ，好的，谢谢回复，发现那个c++的infer以后，panelty结果处理效率好慢啊，竟然比python的慢那么多

**Comment by HongShiKuan** [comment]
请问有人解决将.pth模型转换为.pt模型了吗?
